cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,307 @@ Cask Data Application Platform Release Notes
+
+- `CDAP-4831 <https://issues.cask.co/browse/CDAP-4831>`__ - Allows user-scoped plugins to
+  surface the correct widget JSON in the Hydrator UI.
+
+- `CDAP-4832 <https://issues.cask.co/browse/CDAP-4832>`__ - Added the ErrorDataset as an
+  option on widget JSON in Hydrator plugins.
+
+- `CDAP-4848 <https://issues.cask.co/browse/CDAP-4848>`__ - Added a new section to the
+  CDAP documentation's Administration Manual on upgrading CDAP.
+
+- `CDAP-4850 <https://issues.cask.co/browse/CDAP-4850>`__ - Documented the steps for
+  upgrading CDAP when the underlying Hbase is upgraded to a version incompatible with the
+  currently-installed version of CDAP.
+
+- `CDAP-4852 <https://issues.cask.co/browse/CDAP-4852>`__ - Documented new steps for
+  upgrading either CDAP or CDH with Cloudera Manager clusters.
+
+- `CDAP-4863 <https://issues.cask.co/browse/CDAP-4863>`__ - Upgraded the version of `Twill
+  <http://twill.incubator.apache.org>`__ in CDAP to version 0.7.0.
+
+- `CDAP-4865 <https://issues.cask.co/browse/CDAP-4865>`__ - Enhanced the CDAP SDK to be
+  able to publish metadata updates to an external Kafka, identified by the configuration
+  property metadata.updates.kafka.broker.list. Publishing can be enabled by setting
+  metadata.updates.publish.enabled to true. Updates are published to the Kafka topic
+  identified by the property metadata.updates.kafka.topic.
+
+- `CDAP-4893 <https://issues.cask.co/browse/CDAP-4893>`__ - Improved the text displayed
+  when deleting a widget in the CDAP UI.
+
+- `CDAP-4902 <https://issues.cask.co/browse/CDAP-4902>`__ - Added an example of using the
+  Error dataset with the Hydrator Python Evaluator to the Hydrator documentation.
+
+Bug Fixes
+---------
+- `CDAP-3901 <https://issues.cask.co/browse/CDAP-3901>`__ - Fixed a problem with remote
+  calls for a namespace existence check in CDAP dataset services.
+
+- `CDAP-4419 <https://issues.cask.co/browse/CDAP-4419>`__ - Fixed a problem with
+  intermittent failure of the workflows, as demonstrated with the CDAP workflow guide.
+
+- `CDAP-4473 <https://issues.cask.co/browse/CDAP-4473>`__ - Fixed a problem with Node.js
+  proxy server and unexpected inputs.
+
+- `CDAP-4476 <https://issues.cask.co/browse/CDAP-4476>`__ - Fixed a problem with switching
+  nodes in the Cask Hydrator UI causing unexpected behaviors on slow machines.
+
+- `CDAP-4613 <https://issues.cask.co/browse/CDAP-4613>`__ - Fixed a problem with tooltips
+  not appearing in Flow and Workflow diagrams displayed in the Firefox browser.
+
+- `CDAP-4689 <https://issues.cask.co/browse/CDAP-4689>`__ - Fixed a problem when using an
+  Hbase sink as one of multiple sinks in a Hydrator pipeline.
+
+- `CDAP-4718 <https://issues.cask.co/browse/CDAP-4718>`__ - Corrected problems with the
+  display of success and error messages in the Hydrator UI.
+
+- `CDAP-4720 <https://issues.cask.co/browse/CDAP-4720>`__ - Fixed an issue where system
```

Comment:
No. It's completely manual. I downloaded all the Issues back at 1pm, and then re-wrote them all to have the a similar tense.